enigma/src/main/resources/tinylog.properties

```diff
@@ -0,0 +1 @@
+writer.format = {date: HH:mm:ss.SSS} [{level}]: {message}
```

Comment:
maybe worth adding async output: https://tinylog.org/v2/configuration/#writing-thread -- idk if enigma logging is ever heavy

the default level is `trace`, dunno if it's worth raising that up to info?